sig/datadog/core/rate_limiter.rbs

```diff
@@ -24,11 +49,12 @@ module Datadog
 
       def increment_conforming_count: () -> untyped
 
-      def should_allow?: (untyped size) -> (false | true)
-      def update_rate_counts: (untyped allowed) -> untyped
```

Comment:
Minor: Wait, doesn't this method still exist? Why remove it?

Reply:
🤔 I ran re-generating via and thought it will generate it well enough 

```
bundle exec rbs prototype rb lib/datadog/core/rate_limiter.rb > sig/datadog/core/rate_limiter.rbs 
```

Reply:
Hmm... That's weird. What version of `rbs` are you running? I just tried it and it kept the function for me... :eyes: 

```
$ bundle exec rbs --version
rbs 3.6.1
```

Reply:
Same version. I will spend a bit more time with RBS then.